Inferences/TheoryInstAndSimp.cpp

```diff
@@ -171,8 +170,8 @@ bool TheoryInstAndSimp::isSupportedFunction(Term* trm) {
   auto sym = env.signature->getFunction(trm->functor());
```

Comment:
Can't add a comment to line 210 of this file, so commenting here.

I don't think we should be using `SubtermIterator` on line 210, but rather `NonVariableNonTypeIterator`

Reply:
Good point, changed it.